### ISSUE TITLE
Lacking documentation | Create a wiki

### ISSUE THREAD

**Issue by develcooking** [tom5454/Toms-Storage#225]
Hello,
your mod is great. The only problem that i have with it is the lacking documentation for example in a wiki.
I used your mod since the 1.17 version for a simple chest storage.
Now I experiment a lot with your mod and others to create simple but effective farms.
Ex. I build a "simple" obsidian farm in the end and used your mod to collect the items that a block breaker (Mod called "Block Breakers" on CurseForge) dropped and move it back to the overworld through a level 4 beacon.

I watched your spotlight video to find out how this works. And watched a lot of other tutorials to use your mod for example one thing that I couldn't find out is how to set a limit of blocks in a system and if more items are in it it would throw them out.
Ex. If my Storage system has 500k stone in it   it should fill anymore than that into a dropper that can drop it into lava.
I don't know if something exists like that because I cant read about it.

In a nutshell what i want to say is please make a wiki where people can look at all the items that exist and how they function. I don't really know what a inventory proxy and than I could find more out about it. 

So I want to encourage you to create a Wiki. I would be here to help you with the knowledge that I have.
